app/dashboard/views.py

```diff
@@ -178,6 +186,13 @@ def new_interest(request, bounty_id):
     except Bounty.DoesNotExist:
         raise Http404
 
+    is_too_many_already_started = bounty.work_scheme == 'traditional' and bounty.interested.filter(pending=False).count() > 0
```

Comment:
We could do:

`is_too_many_already_started = bounty.work_scheme == 'traditional' and bounty.interested.filter(pending=False).exists()`
to avoid the `COUNT()`.

Also, can we add `Bounty.SCHEME_TYPES` and the individual scheme types, so: `bounty.work_scheme == 'traditional'` would become:  `bounty.work_scheme == Bounty.SCHEME_TRADITIONAL` or something like that.